tox.ini

```diff
@@ -0,0 +1,70 @@
+[tox]
+envlist =
+    lint,
+    test
+
+[testenv]
+basepython = python3
+
+[testenv:test]
+deps =
+    pytest
+    pytest-girder
+commands =
+    pytest {posargs}
+
+[testenv:lint]
+skipsdist = true
+skip_install = true
+deps =
+    flake8
+    flake8-bugbear
+    flake8-docstrings
+    flake8-quotes
+    pep8-naming
+commands =
+    flake8 {posargs}
+
+[testenv:release]
+passenv =
+    TWINE_USERNAME
+    TWINE_PASSWORD
```

Comment:
I like to also add `TWINE_REPOSITORY_URL`, then can test the first release to test.pypi.com by configuring the appropriate value.

Reply:
See 19118408453d7a5ccbceffdfd146f362c8c32418